worker/dbaccessor/tracker_test.go

```diff
@@ -369,7 +358,7 @@ func (s *trackedDBWorkerSuite) TestWorkerCancelsTxn(c *gc.C) {
 		case <-time.After(testing.ShortWait):
 			c.Fatal("timed out waiting for sync")
 		}
-		workertest.CheckKill(c, w)
+		_ = workertest.CheckKill(c, w)
```

Comment:
This should be DirtyKill if we don't care about the error, if we do, we should use `CleanKill`.

```suggestion
		workertest.DirtyKill(c, w)
```